### PR TITLE
Add hacky script to get medallia script on allowed staging TeamSite pages

### PR DESCRIPTION
**Pull request by billfienberg**
## GitHub Issue

https://github.com/department-of-veterans-affairs/va.gov-team/issues/18319

## Description

The Office of Resolution Management, Diversity & Inclusion (ORMDI) pages live in Enterprise Web Infrastructure Support (EWIS). EWIS doesn't have different environments, like prod and staging, so we need to use prod. We need to load this survey on staging, but not prod. Therefore, I added a new script that does the following:

1. returns early if the `host` doesn't include `'staging'`
1. returns early if the `pathname` isn't included in an allowlist
1. create and append the Medallia script

## Acceptance criteria
- [ ] The 9-question Medallia survey (like the one that pops up on staging.va.gov/find-locations/facility/vha_691) shows up on the following pages:

- https://www.staging.va.gov/ORMDI/
- https://www.staging.va.gov/ORMDI/NoFEAR_Select.asp
- https://www.staging.va.gov/ORMDI/Contact_Us.asp
- https://www.staging.va.gov/ORMDI/EEOcomplaint.asp
- https://www.staging.va.gov/ORMDI/HPP.asp
- https://www.staging.va.gov/ORMDI/Diversity_Inclusion.asp
- https://www.staging.va.gov/ORMDI/Reasonable_Accommodations1.asp
- https://www.staging.va.gov/adr/

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

## Links

- [ORMDI pages are `.asp`](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18319#issuecomment-770993435)
- [Both the staging and prod versions of /ORMDI are using the production Foresee script](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18319#issuecomment-771093933)
- [EWIS Notes](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18319#issuecomment-771160161)
- [EWIS uses `vagovprod`](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18319#issuecomment-771173347)
- [Using `targetEnvironment` query string on TeamSite pages](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18319#issuecomment-771220859)

### CODE REVIEW

**Review by timwright12 — APPROVED** [review]

File: src/site/assets/js/medallia/vagovprod.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Medallia Embed Script
+(function(g) {
+  var isStaging = window.location.host.includes('staging');
+  if (!isStaging) return
```
Comment (ncksllvn) [inline src/site/assets/js/medallia/vagovprod.js:4]: What if it's dev or localhost - is this okay to have execute?

**Review by ncksllvn — APPROVED** [review]

File: src/site/assets/js/medallia/vagovprod.js (hunk 1 above)
Comment (billfienberg) [reply src/site/assets/js/medallia/vagovprod.js:4]: I think we only want the survey on staging. Eventually, we will want it on prod. 

For example, a different Medallia survey gets loaded on staging.va.gov/find-locations, but not dev.va.gov/find-locations.

image

Comment (ncksllvn) [reply src/site/assets/js/medallia/vagovprod.js:4]: Oh yeah I read this wrong. Makes sense. Thanks!

**Comment by billfienberg** [comment]
I just got off a call, and it sounds like Brendan F. will be adding the Medallia script directly to TeamSite. So I'm going to revert this PR.

**Pull request merged**